speech/tests/system.py

```diff
@@ -46,6 +47,9 @@ def _wait_until_complete(operation, max_attempts=10):
     :rtype: bool
     :returns: Boolean indicating if the operation is complete.
     """
+    # This bizarre delay is necessary because the v1 API seems to return
+    # the v1beta1 type URL sometimes if you poll too soon.
+    time.sleep(3)
```

Comment:
Can we add a tracking issue (or at least a calendar event) to roll this back in 7 days?